chart/epinio-ui/Chart.yaml

```diff
@@ -15,4 +15,4 @@ name: epinio-ui
 sources:
 - https://github.com/epinio/ui
 type: application
-version: 0.4.0
+version: 0.5.0
```

Comment:
any reason why this shouldn't be `v1.0.0-rc1` ?

Reply:
We would need to remove the `rc1` when epinio 1.0 is released. I was pondering what to do generally about UI RC labelling, in the end stuck with the existing pattern.